### DISCUSSION TITLE
Card YAML Info On The Same Page

### DISCUSSION THREAD

**Discussion by SlRvb** [mgmeyers/obsidian-kanban#10] [Ideas & Feature Requests]
For those who don't want to create new pages but still want the use the YAML idea from #6; I think some way for them to add "YAML" info into their card would be really useful.

Since the cards use `- [ ]` I was thinking that maybe regular bullet points could be how other info is stored?

#### Markdown:
![Obsidian_XV9eDVn7OA](https://user-images.githubusercontent.com/54087190/115780653-44be8a00-a36e-11eb-96f9-802895d4b8ee.png)
#### Result:
![Designer_3MgSCLa5CS](https://user-images.githubusercontent.com/54087190/115783628-f317fe80-a371-11eb-8e4e-64f8c5aedd39.png)

- A regular bullet point would just appear as regular text in the card
- Using the same format as Dataview for inline yaml could be helpful which would allow that info to be toggled/sorted.

**Comment by mgmeyers** [comment]
Ah, interesting! Do you know what happens if you have multiple of a single inline YAML key? Like, if two cards had `Description:: ...` Would that mess with thing like dataview?

**Reply by SlRvb** [reply]
Seems like Dataview only grabs the last one with that key.

**Comment by mgmeyers** [comment]
Also, one of the additional output formats I was thinking was something like this, where you could have a lot more detail per card (essentially any markdown between level three headers):

```markdown
## Lane one

### This is a card with a description and subtasks
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum a ligula vitae massa vehicula commodo. Phasellus sapien nibh, molestie sed congue in, tincidunt pellentesque lectus.

#tag1 #tag2

- [ ] molestie sed congue in
- [ ] Phasellus sapien nibh
- [ ] consectetur adipiscing elit



## Done lane
**Complete**

### This is a completed task
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum a ligula vitae massa vehicula commodo. Phasellus sapien nibh, molestie sed congue in, tincidunt pellentesque lectus.

#tag3 #tag4

- [x] molestie sed congue in
- [x] Phasellus sapien nibh
- [x] consectetur adipiscing elit
```

**Reply by SlRvb** [reply]
I think this would be pretty great! I hope that a Heading 1 could be the title of the Kanban view.

I think my only question with this approach is how will you get the same kind of metadata for the subtasks? 🤔
